.travis.yml

```diff
@@ -5,6 +5,7 @@ branches:
 language: java
 jdk:
 - oraclejdk8
+dist: trusty
```

Comment:
This change is required since the old Xenial build environment  in Travis stops supporting  pre-install OracleJDK8 now.  It is suggested that if OracleJDK is needed that we need to switch to Trusty build env for building.  See following for detail:
https://travis-ci.community/t/the-travis-ci-build-could-not-be-completed-due-to-an-error/1345